src/FS.Embedded.Manifest.Task/build/netstandard2.0/Microsoft.Extensions.FileSystem.Embedded.Manifest.Task.targets

```diff
@@ -0,0 +1,44 @@
+<Project>
+
+  <UsingTask
+    AssemblyFile="Microsoft.Extensions.FileSystem.Embedded.Manifest.Task"
+    TaskName="Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.GenerateEmbeddedResourcesManifest" />
+
+  <PropertyGroup>
+    <EmbeddedFilesManifest Condition="$(EmbeddedFilesManifest) == ''">false</EmbeddedFilesManifest>
+    <EmbeddedFilesManifestFileName Condition="$(EmbeddedFilesManifestFileName) == ''">Microsoft.Extensions.FileSystem.Embedded.Manifest.xml</EmbeddedFilesManifestFileName>
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(EmbeddedFilesManifest) == 'true'">
+    <EmbeddedResource
+      Include="$(IntermediateOutputPath)$(EmbeddedFilesManifestFileName)"
+      IsManifest="true"
+      Condition="@(EmbeddedResource->WithMetadataValue('IsManifest','true')->Count()) == 0"
+      LogicalName="$(EmbeddedFilesManifestFileName)" />
+  </ItemGroup>
+
+  <Target 
+    Name="GenerateEmbeddedFilesManifest"
+    AfterTargets="PrepareResourceNames"
+    Condition="$(EmbeddedFilesManifest) == 'true'"
+    Inputs="@(EmbeddedResource)"
```

Comment:
Unfortunately this is going to run in to https://github.com/Microsoft/msbuild/issues/2470. csc works around this by manually tracking file hashes - https://github.com/Microsoft/msbuild/pull/1328/files